util/importers/LDC97S62.py

```diff
@@ -94,7 +102,7 @@ def _populate_batch_queue(self, session):
                     self._x_length: source_len,
                     self._y: target,
                     self._y_length: target_len})
-            except (RuntimeError, tf.errors.CancelledError):
+            except Exception as e:
```

Comment:
Why is this required? I once tried to make it as specific as possible. Now it captures everything and by that it'd make debugging much harder.